docs/8-veba-troubleshooting.md

```diff
@@ -0,0 +1,277 @@
+```
+[{
+                "type": "stream",
+                "provider": "vmware_vcenter",
+                "address": "https://vc01.labad.int/sdk",
+                "auth": {
+                        "method": "user_password",
+                        "secret": {
+                                "username": "administrator@vsphere.local",
+                                "password": "WrongPassword"
+                        }
+                },
+                "options": {
+                        "insecure": "true"
+                }
+        },
+
+```
+
+We now fix the Kubernetes configuration with 3 commands - delete and recreate the secret file, then delete the broken pod. Kubernetes will automatically spin up a new pod with the new configuration. We need to do this because the JSON configuration file is not directly referenced by the event router. The JSON file is mounted into the event router pod as a Kubernetes secret. 
+
+```
+kubectl -n vmware delete secret event-router-config
+kubectl -n vmware create secret generic event-router-config --from-file=event-router-config.json
+kubectl -n vmware delete pod vmware-event-router-5dd9c8f858-5c9mh 
+```
+
+We get a pod list again to determine the name of the new pod
+```
+kubectl get pods -A
+```
+
+Here is the command output:
+```
+NAMESPACE        NAME                                   READY   STATUS        RESTARTS   AGE
+kube-system      coredns-584795fc57-hcvxh               1/1     Running       1          4d19h
+kube-system      coredns-584795fc57-hf72w               1/1     Running       1          4d19h
+kube-system      etcd-veba02                            1/1     Running       1          4d19h
+kube-system      kube-apiserver-veba02                  1/1     Running       1          4d19h
+kube-system      kube-controller-manager-veba02         1/1     Running       1          4d19h
+kube-system      kube-proxy-fj47p                       1/1     Running       1          4d19h
+kube-system      kube-scheduler-veba02                  1/1     Running       1          4d19h
+kube-system      weave-net-vs8ls                        2/2     Running       4          4d19h
+projectcontour   contour-5cddfc8f6-8hzd6                1/1     Running       1          4d19h
+projectcontour   contour-5cddfc8f6-jq7d8                1/1     Running       1          4d19h
+projectcontour   contour-certgen-f92l5                  0/1     Completed     0          4d19h
+projectcontour   envoy-gcmqt                            1/1     Running       1          4d19h
+vmware           tinywww-7fcfc6fb94-mfltm               1/1     Running       1          4d19h
+vmware           vmware-event-router-5dd9c8f858-5c9mh   0/1     Terminating   40         3h9m
+vmware           vmware-event-router-5dd9c8f858-wt64s   1/1     Running       0          28s
+```
+
+Now view the event router logs:
+```
+kubectl logs -n vmware vmware-event-router-5dd9c8f858-wt64s
+
```

Comment:
Can we get rid of these empty lines in command blocks?

Reply:
Fixed